analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6-SPDX2TV.spdx

```diff
@@ -0,0 +1,5693 @@
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception-2.0
+FileCopyrightText: <text> Copyright (C) 2011-2020 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: alsa-utils-1.2.6.tar.bz2/alsa-utils-1.2.6.tar/alsa-utils-1.2.6/configure
+SPDXID: SPDXRef-item441857
+FileChecksum: SHA1: 54757072b22a16e19f63b27eb9e5666af411f960
+FileChecksum: SHA256: f7f5845f92e0bba08a15a281082c2003319551b8e20b1ef0c1fb527144905db5
+FileChecksum: MD5: 60e1188ca4a68f2ec73b35078fe24a95
+LicenseConcluded: LicenseRef-FSFUL
+LicenseInfoInFile: LicenseRef-FSFUL
+LicenseInfoInFile: LicenseRef-fsf-free
+FileCopyrightText: <text> Copyright (C) 1992-1996, 1998-2012 Free Software Foundation, Inc.
+Copyright (C) 2012 Free Software Foundation, Inc. 
+Copyright (C) 2012 Free Software Foundation, Inc.  </text>
+
+##File
+
+FileName: alsa-utils-1.2.6.tar.bz2/alsa-utils-1.2.6.tar/alsa-utils-1.2.6/config.guess
+SPDXID: SPDXRef-item441763
+FileChecksum: SHA1: 5e7859a1f79e6ea39193b3409c2f9a32628567b7
+FileChecksum: SHA256: c02eb9cc55c86cfd1e9a794e548d25db5c9539e7b2154beb649bc6e2cbffc74c
+FileChecksum: MD5: 367ad42b8ced25b62e67886d73730e15
+LicenseConcluded: LicenseRef-GPL-3.0-or-later-WITH-Autoconf-exception-generic-3.0-0d71aa9829a4d0c69a85a5055efb5613
+LicenseInfoInFile: LicenseRef-GPL-3.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic-3.0
+LicenseInfoInFile: LicenseRef-free-unknown
+LicenseInfoInFile: LicenseRef-gpl-3.0-plus
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception
+FileCopyrightText: <text> Copyright 1992-2018 Free Software Foundation, Inc.
+Copyright 1992-2018 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: alsa-utils-1.2.6.tar.bz2/alsa-utils-1.2.6.tar/alsa-utils-1.2.6/config.sub
+SPDXID: SPDXRef-item441935
+FileChecksum: SHA1: acb5dfe011a26adbf9e43e0279166dd412f2a830
+FileChecksum: SHA256: 8dd30cd3972eec02262c317244a9e38ccdf5a0d7fc9eb598edd7e861824cdf8e
+FileChecksum: MD5: 13706dab007cfebc0b59d5fe6ba86a4c
+LicenseConcluded: LicenseRef-GPL-3.0-or-later-WITH-Autoconf-exception-generic-3.0-0d71aa9829a4d0c69a85a5055efb5613
+LicenseInfoInFile: LicenseRef-GPL-3.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic-3.0
+LicenseInfoInFile: LicenseRef-free-unknown
+LicenseInfoInFile: LicenseRef-gpl-3.0-plus
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception
+FileCopyrightText: <text> Copyright 1992-2018 Free Software Foundation, Inc.
+Copyright 1992-2018 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: alsa-utils-1.2.6.tar.bz2/alsa-utils-1.2.6.tar/alsa-utils-1.2.6/alsaucm/usecase.c
+SPDXID: SPDXRef-item441945
+FileChecksum: SHA1: 058b0cd5d4ffbd20db070f611efa27c3f2fa5ab9
+FileChecksum: SHA256: 949d5690c26dd5a39210eb41fe8cf5c7eca362ec353a03802b6c49a955187a1c
+FileChecksum: MD5: 6e60b1ca0197ee76b7d0ba8079900378
+LicenseConcluded: LicenseRef-LGPL-2.0-or-later-c4a20a7c9332c01337a88843c850c213
```

Comment:
E.g. here you have only the boilerplate of LGPL-2.0-or-later as license text conlude, please correct this. There might ba also other files with the wrong license text